assets/js/translator.js

```diff
@@ -730,6 +744,115 @@ function translateDoc() {
     }
 }
 
+function translateLink(href) { // eslint-disable-line no-unused-vars
+    $('#webpage').val(href);
+    translateWebpage();
+}
+
+function cleanPage(html) {
+    // Pages like
+    // eslint-disable-next-line max-len
+    // http://www.lapinkansa.fi/sagat/romssa-sami-searvvi-jodiheaddji-daruiduhttinpolitihkka-vaikkuha-ain-olbmuid-guottuide-samiid-birra-15843633/
+    // insert noise using document.write that 1. makes things enormously slow, and 2. completely mess up styling so e.g. you
+    // have to scroll through a full screen of whitespace before reaching content. This might mess things up some places – needs
+    // testing – but on the other hand most uses of document.write are evil.
+    return html.replace(/document[.]write[(]/g,
```

Comment:
Um, I think I am not sure what I am supposed to do here :/ Should I reduce the cluttering of the comments (by spanning them across more number of lines). Please pardon for a silly question here :P